nilearn/decoding/decoder.py

```diff
@@ -106,41 +107,131 @@ def _check_param_grid(estimator, X, y, param_grid=None):
 
     """
     if param_grid is None:
+        param_grid = _default_param_grid(estimator, X, y)
+
+    else:
+        if isinstance(estimator, (RidgeCV, RidgeClassifierCV)):
+            param_grid = _wrap_param_grid(param_grid, "alphas")
+
+    return param_grid
+
+
+def _default_param_grid(estimator, X, y):
+    """Generate sensible default for param_grid.
+
+    Parameters
+    ----------
+    estimator: str, optional
+        The estimator to choose among:
+        %(classifier_options)s
+        %(regressor_options)s
+        Default 'svc'.
```

Comment:
There is no default (and there should not be any I guess).

Reply:
Good catch, removing the default from the docstring